### PR TITLE
Add an rstcheck project

### PR DESCRIPTION
**Pull request by myint**
This is as discussed over email with @sils.

### CODE REVIEW

File: data/projects/rstcheck_with_better_sphinx_support.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+* Be familiar what `rstcheck` does and how it does it.
+* Be familiar with how to extend Sphinx. Particularly, learn how to parse code
+  blocks.
+
+##### GSOC 2017 MIDTERM
+
+* Create a basic front end, `rstcheck-sphinx`, that does `sphinx-build`-style
+  parsing. This differentiates it from `rstcheck` that uses plain `docutils`
+  only minor modification to somewhat work with Sphinx.
```
Comment (sils) [inline data/projects/rstcheck_with_better_sphinx_support.md]: can you clarify? I'm not sure I fully understand what those modifications are and I fear that might have a negative effect on the applications we get on this.

File: data/projects/rstcheck_with_better_sphinx_support.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+##### GSOC 2017 FINAL
+
+* Write scripts to automatically test against existing Sphinx projects to check
+  for false positives in `rstcheck-sphinx`.
+* Integrate with the main `rstcheck` script if appropriate.
```
Comment (sils) [inline data/projects/rstcheck_with_better_sphinx_support.md]: IMO it'd be nice to have that integrated into coala as well, that won't take a long time at all and it's something we have been looking for anyway. Changing the integration to use the new stuff should be like a days work or so.

File: data/projects/rstcheck_with_better_sphinx_support.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+* `rstcheck` = `docutils` + code block linting
+* `rstcheck-sphinx` = `sphinx-build` + code block linting
+
+#### Milestones
+
+##### GSOC 2017 CODING STARTS
+
+* Be familiar what `rstcheck` does and how it does it.
+* Be familiar with how to extend Sphinx. Particularly, learn how to parse code
+  blocks.
```
Comment (myint) [inline data/projects/rstcheck_with_better_sphinx_support.md]: Can you clarify what the "CODING STARTS" section means? Based on one of the other examples I looked at, it seemed like prerequisites.

File: data/projects/rstcheck_with_better_sphinx_support.md (hunk 2 above)
Comment (myint) [inline data/projects/rstcheck_with_better_sphinx_support.md]: Added.

File: data/projects/rstcheck_with_better_sphinx_support.md (hunk 3 above)
Comment (sils) [inline data/projects/rstcheck_with_better_sphinx_support.md]: yes, thanks for the suggestion. WIll make an issue

**Comment by sils** [comment]
ack 60e4f826dbc90008aa9f8d4135deb0b44c2c64c8 61b85224ed03e8aa80a74271a1631bf7a7603356 11bf0a66ea900e3383ea15e6117bdfe3187f5465 1bbec82e437a2a9352713bf7316ead641788eaf6 63fc50e71e75c80225d29c70b5929563a4da44a1 54bf849030303cef0ddba3783bdb1192ba6fd332 f084b141565e4557d9c50d92e81ebbc86f4af3ca

**Comment by sils** [comment]
@rultor merge

**Pull request merged**